reference/api/overview.md

```diff
@@ -1,45 +1,115 @@
 # API reference
 
-Welcome to the Meilisearch API documentation.
+Welcome to the Meilisearch API documentation. If you are new to Meilisearch, check out our [quick start guide!](/learn/getting_started/quick_start.md)
 
-::: tip
+Meilisearch is a RESTful API. This page describes the general behavior of the API.
 
-Check out [the FAQ](/resources/faq.md) for answers to some common questions 💡
+### OpenAPI
 
+The Meilisearch OpenAPI specifications:
+
+- [YAML](https://bump.sh/doc/meilisearch.yaml)
+- [JSON](https://bump.sh/doc/meilisearch.json)
+
+## Document conventions
+
+The API documentation uses the following conventions:
+
+- Curly braces (`{}`) in API routes represent path parameters, e.g., GET `/indexes/{index_uid}`
+- Required fields are marked by an asterisk (`*`)
+- Placeholder text is in uppercase characters with underscore delimiters, e.g., `MASTER_KEY`
+
+## Authorization
+
+By [providing Meilisearch with a master key at launch](/learn/security/master_api_keys.md#protecting-a-meilisearch-instance), you protect your instance from unauthorized requests. From then on, you must include the `Authorization` header along with a valid API key to access protected routes (all routes except `/health`).
+
+<CodeSamples id="authorization_header_1" />
+
+The [`/keys`](/reference/api/keys.md) route can only be accessed using the master key. For security reasons, we recommend using regular API keys for all other routes.
+
+::: note
+ v0.24 and below use the `X-MEILI-API-KEY: apiKey` authorization header:
+<CodeSamples id="updating_guide_check_version_old_authorization_header" />
 :::
 
+[To learn more about keys and security, refer to our dedicated guide.](/learn/security/master_api_keys.md)
+
+## Pagination
+
+Meilisearch paginates all GET routes that return multiple resources, e.g., GET `/indexes`, GET `/documents`, GET `/keys`, etc. This allows you to work with manageable chunks of data. All these routes return 20 results per page, but you can configure it using the `limit` query parameter. You can move between pages using `offset`.
+
+All paginated responses contain the following fields:
+
+| Name         | Type    | Description                  |
+| :----------- | :------ | :--------------------------- |
+| **`offset`** | Integer | Number of resources skipped  |
+| **`limit`**  | Integer | Number of resources returned |
+| **`total`**  | Integer | Total number of resources    |
+
+### `/tasks` endpoint
+
+Since the `/tasks` endpoint uses a different type of pagination, the response contains different fields. You can read more about it in the [tasks API reference](/reference/api/tasks.md#get-tasks).
+
+## Parameters
+
+Parameters are options passed to the endpoint to influence the response. They are added at the end of the string for GET requests. For POST requests, they are added to the body.
```

Comment:
Ok, this is a big edit in multiple parts. Parameters can be sort of confusing, so I hope I'm not saying anything wrong 😅 

I think it's confusing to describe query parameters (`http://example.com?a=b&c=d`) both here, mixed with request body parameters, and below, under `### Query parameters`.

My suggestions are:
- use this paragraph to list the types of parameters we use in Meilisearch's API, but refrain from explaining them;
- create a `### Request body parameters` for POST/PATCH/PUT requests
- move the stuff about GET requests here to `### Query parameters`

```suggestion
Parameters are options you can pass to an API endpoint to modify its response. There are three main types of parameter in Meilisearch's API: request body parameters, path parameters, and query parameters.

### Request body parameters

These parameters are mandatory parts of POST, PUT, and PATCH requests. They accept a wide variety of values and data types depending on the resource you're modifying. You must add these parameters to your request's data payload.

```